softgroup/services/messenger/messenger-api/src/main/java/com/softgroup/messenger/api/dto/enumeration/ConversationType.java

```diff
@@ -1,8 +1,16 @@
 package com.softgroup.messenger.api.dto.enumeration;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
 /**
  * Created by user on 26.03.2017.
  */
 public enum ConversationType {
-    INDIVIDUAL,GROUP;
+    @JsonProperty("0")
+    INDIVIDUAL,
+    @JsonProperty("1")
```

Comment:
@Kun-Kun No need to add bunch of this JsonProperty annotations. Make enum with constructor and use annotations JsonCreator and JsonValue